test/quantization/fx/test_quantize_fx.py

```diff
@@ -5617,6 +5617,15 @@ def forward(self, x):
                 out = model_quantized(input.to(device_after))
                 self.assertEqual(out.device.type, device_after)
 
+    def _test_model_dropout(self):
```

Comment:
nit: this looks like a normal test instead of a helper function, probably don't need to start with `_`

Reply:
I think we can add another similar test in eager mode test as well: https://github.com/pytorch/pytorch/blob/master/test/quantization/eager/test_quantize_eager_qat.py#L246